pkgs/development/python-modules/mailman-rss/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonApplication, fetchPypi
+, dateutil, future, requests, beautifulsoup4 }:
+
+buildPythonApplication rec {
+  pname = "mailman-rss";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1brrik70jyagxa9l0cfmlxvqpilwj1q655bphxnvjxyganxf4c00";
+  };
+
+  propagatedBuildInputs = [ dateutil future requests beautifulsoup4 ];
+
+  doCheck = false;
```

:smile: I always forgot the comment
```suggestion
  # No tests in Pypi Tarball
  doCheck = false;
```